stock_quant_manual_assign/README.rst

```diff
@@ -30,7 +30,7 @@ help us smashing it by providing a detailed and welcomed `feedback
 <https://github.com/OCA/
```

Comment:
up: 
"With this module, user can manually change the automatic quant selection."
and
"How to use this module:

# Select the quants from each stock.move. 
# A button opens a wizard directly from the move or from the picking."